presto-main-base/src/test/java/com/facebook/presto/sql/query/QueryAssertions.java

```diff
@@ -142,4 +142,15 @@ public void close()
     {
         runner.close();
     }
+
+    protected void executeExclusively(Runnable executionBlock)
+    {
+        runner.getExclusiveLock().lock();
+        try {
+            executionBlock.run();
+        }
+        finally {
+            runner.getExclusiveLock().unlock();
+        }
+    }
```

Comment:
Can we not re-use this method inside of `TestAccessControlFiltersMasks`?

Reply:
Not really, in `TestAccessControlFiltersMasks` we are asserting the plan, so `QueryAssertions` isn't used. It could be possible to put in `LocalQueryRunner`, which is used by all tests but this is really to do with a test fixture and not so much a query runner. It's just a few lines, so I figured it's least disruptive to copy in  `TestAccessControlFiltersMasks`.

Reply:
Since multiple masks are not allowed on a column, this test now throws an exception. There aren't any other tests that check this behavior, so I'll convert it to verify that it throws.